doc/design/speech/README.MD

```diff
@@ -136,7 +136,23 @@ CTC-Beam search                         | Not supported yet.
 
 ### Row Convolution
 
-TODO by Assignees
+- Difference with sequence convolution
+
+  The 1D sequence convolution (`paddle.layer.sequence_conv`) was supported, but this row convolution is different from it as shown in figure 2 and 3. In the figures, assumed the context length is 3 and the hidden size of each time step is `d`. The weight dimension of sequence convolution is `3d x d`. And weight dimension of row convolution is `3 x d`. The weights are shared among all time steps.
```

Comment:
the first sentence:   row convolution(`paddle.layer.row_conv`) is performed upon 1D sequence input, but it is different from 1D sequence convolution (`paddle.layer.sequence_conv`). The difference is shown in figure 2 and figure 3.